storagenode/gracefulexit/chore.go

```diff
@@ -70,16 +79,16 @@ func (chore *Chore) Run(ctx context.Context) (err error) {
 
 		for _, satellite := range satellites {
 			satelliteID := satellite.SatelliteID
-			worker := NewWorker(chore.log, chore.satelliteDB, satelliteID)
+			worker := NewWorker(chore.log, chore.store, chore.satelliteDB, chore.trust, chore.dialer, satelliteID)
```

Comment:
just curious what's the reason for passing in the `trust` down to worker instead of getting the address here and then pass the address down to the worker

Reply:
Yeah I agree. Only the address is necessary. Updated